.travis.yml

```diff
@@ -1,5 +1,63 @@
-language: node_js
-node_js:
-  - "node"
+sudo: false
+
+language:
+  - php
+  - node_js
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+branches:
+  only:
+    - master
+
+cache:
+  directories:
+    - vendor
+    - $HOME/.composer/cache
+
+matrix:
+  include:
+    - php: 7.1
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: WP_VERSION=latest
+    - php: 5.2
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: TRAVISCI=phpcs
+    - env: TRAVISCI=js
```

Comment:
You still need a "language" to form the base of the build job in the matrix I believe, each job in the matrix should have both a language for the job i.e. `- php: 7.1` and its associated environment matrix ` -env: TRAVISCI =js`

_If Travis were running jobs against this PR we could take a look_ 😖